tests/test_utils.py

```diff
@@ -45,3 +46,22 @@ def test_get_best_ranks(n, best):
     ranks = FeatureRanks(features=[5, 0, 1, 4], ranks=[1, 2, 3, 4], n_feats=10)
     best_feats = utils.get_best_n_features(ranks, n)
     assert sorted(best_feats) == sorted(best)
+
+
+def test_compute_t_student_p_value():
+    """Using p-values tables for t-student with 17 dof"""
+    dof = 17
+    n = dof + 1
+    random_state = np.random.RandomState(9)
+    sample_005 = -1.74
+    sample_01 = -1.333
+    p_values_005 = []
+    p_values_01 = []
+    for i in range(1000):
+        population = random_state.normal(size=n)
+        p_value = utils.compute_t_student_p_value(sample_005, population)
+        p_values_005.append(p_value)
+        p_value = utils.compute_t_student_p_value(sample_01, population)
+        p_values_01.append(p_value)
+    assert abs(np.mean(p_values_005) - 0.05) < 0.01
```

Comment:
What is the likelihood of this failing? Ideally it should be <0.1%, else you might want to ramp up the samples or relax the asserts below.

Reply:
the likelihood should be 0, because the random state is fixed